nilearn/regions/tests/test_parcellations.py

```diff
@@ -136,13 +140,20 @@ def test_parcellations_transform_single_nifti_image(
     assert signals.shape == (test_image_2.shape[3], n_parcel)
 
 
+def test_parcellations_transform_verbose(test_image_2):
+    """Test verbose mostly for coverage purpose."""
+    parcellator = Parcellations(method="kmeans", n_parcels=5, verbose=1)
```

Comment:
I don't remember what values for verbose make sense for Parcellations. Sometimes you can pas higher values to get more details but sometimes verbose is equivalent to a boolean. It could be the opportunity to test different input values here and check that things are not crashing...

Reply:
Did a quick check, but will test.

I mostly did that because of all the noise while testing because Parcellations has default verbosity of 1.